go/test/endtoend/vtgate/plan_tests/main_test.go

```diff
@@ -128,6 +129,31 @@ func start(t *testing.T) (utils.MySQLCompare, func()) {
 	}
 }
 
+// parseSQL statements - querySQL may be a multi-line sql blob
+func parseSQL(querySQL ...string) ([]string, error) {
```

Comment:
in this code base, `parse` means something pretty specific, and this is not it :)

maybe call it `splitSQL` instead?

Reply:
thanks , renamed the function with 7ac0eb2c002e364991bcf3b9630ad58be407d16e